crates/chain/tests/test_keychain_txout_index.rs

```diff
@@ -89,7 +89,7 @@ fn append_changesets_check_last_revealed() {
         keychains_added: BTreeMap::<(), _>::new(),
         last_revealed: rhs_di,
     };
-    lhs.append(rhs);
+    lhs.merge(rhs);
```

Comment:
@LagginTimes nit: should we rename the test fn name ? wdyt ?

Reply:
A few spots in this file to rename append -> merge and then should be ready to go.
line 55: in the docs
line 61: test name
line 683: function wasn't renamed, don't know why not caught by CI